source/adminguide/hosts.rst

```diff
@@ -207,6 +207,22 @@ any system that is not up to date with patches.
 <http://docs.cloudstack.org/Knowledge_Base/Possible_VM_corruption_if_XenServer_Hotfix_is_not_Applied/Highly_Recommended_Hotfixes_for_XenServer_5.6_SP2>`_.
 
 
+Hypervisor Capabilities
+-----------------------
+For different hypervisors and their versions, various capabilities such as maximum number of guest VMs per host, maximum number of volumes per VM, security group support, etc are considered by CloudStack. These capabilities are stored in the **cloud.hypervisor_capabilities** table in the database. If a specific hypervisor version is not available in the database, values against the *default* version for the hypervisor will be used.
+
+Following hypervisor-specific documentations can be referred for different maximums for a particular hypervisor host:
+
+- VMware: `VMware Configuration Maximum tool <https://configmax.vmware.com/guest?vmwareproduct=vSphere&release=vSphere%207.0&categories=1-0,2-0>`_.
+
+- Citrix Hypervisor/Xenserver/XCP-ng: `Configuration limits | Citrix Hypervisor 8.2 <https://docs.citrix.com/en-us/citrix-hypervisor/system-requirements/configuration-limits.html>`_.
+
+
+.. note::
+   Guest VM limit check is not done while deploying a VM on a KVM hypervisor host.
+
```

Comment:
The number of guest VMs can be updated using _updateHypervisorCapabilities_ API. @shwstppr  Do you mentioned it, and about other APIs for hypervisor capabilities (_listHypervisorCapabilities_, _updateHypervisorCapabilities_).

Reply:
@sureshanaparti as you can see in diff I've not mentioned APIs. I'll make the changes